tests/test_job.py

```diff
@@ -73,7 +75,7 @@ def testInitWithoutTemplate():
     """Test ParslJob stdout/stderr init. with template."""
     # Test doesn't actually use directory
     submit_path = os.path.join(TESTDIR, "without_template")
-    config = BpsConfig({"subDirTemplate": "", "submitPath": submit_path})
+    config = BpsConfig({"subDirTemplate": "", "submitPath": submit_path}, search_order=[])
```

Comment:
If this works then merge it, but what's the default search order that this is setting to the empty list?

Reply:
The default search order is defined [here](https://github.com/lsst/ctrl_bps/blob/main/python/lsst/ctrl/bps/bps_config.py#L51) and it controls the order in which the config's sections are being check for the existence of the search key. However, it was only used when a `BpsConfig` was read from a file and it was an empty list when a `BpsConfig` was created out of a dictionary (as these tests do) or `lsst.daf.butler.Config`. DM-44110 changed this behavior. Now it is always set to the default order unless specified explicitly.